lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -18,9 +18,18 @@
 #include "Library/ForInObjectEnumerator.h"
 #include "Library/EngineInterfaceObject.h"
 #include "Library/IntlEngineInterfaceExtensionObject.h"
+#include "Library/JsBuiltInEngineInterfaceExtensionObject.h"
 #include "Library/ThrowErrorObject.h"
 #include "Library/StackScriptFunction.h"
 
+#include "ByteCode/ByteCodeSerializer.h"
+
+#pragma warning(push)
+#pragma warning(disable:4309) // truncation of constant value
+#pragma warning(disable:4838) // conversion from 'int' to 'const char' requires a narrowing conversion
+
+#pragma warning(pop)
```

Comment:
Is this intentional? it looks like the warning setup for including the bytecode, but as far as I know this is a no-op right now?

Reply:
Inherited this from the builtins branch and forgot to clean it up! Thanks.